src/taskchampion-cpp/Cargo.toml

```diff
@@ -10,6 +10,8 @@ crate-type = ["staticlib"]
 [dependencies]
 taskchampion = "0.7.0"
 cxx = "1.0.124"
+# Not used directly, but adds the native-roots feature. #3625.
+reqwest = { version = ">= 0.11.27", default-features = false, features = ["rustls-tls-native-roots"] }
```

Comment:
This "fake dependency" is probably not the best way to solve the issue. Probably, we should mirror this feature from taskchampion into reqwest (on which it depends). But this will do for testing.